src/ErrorData.ts

```diff
@@ -10,7 +10,9 @@ export class ErrorData {
     constructor(error: any) {
         if (error instanceof Error) {
             try {
+                // tslint:disable-next-line:no-unsafe-any

```

Comment:
You can remove this entire file if you want. I should've removed it in this commit, but I forgot: https://github.com/Microsoft/vscode-azureappservice/commit/3e1c8a665840537a5a0de7512ecccbc863d2e055